src/cirq_iqm/iqm_sampler.py

```diff
@@ -23,12 +23,13 @@
 import numpy as np
 from cirq import study
 from cirq.study import resolver
+from iqm_client.iqm_client import Circuit, IQMClient, SingleQubitMapping
+
 from cirq_iqm import IQMDevice
-from cirq_iqm.iqm_client import CircuitDTO, IQMClient, SingleQubitMappingDTO
 from cirq_iqm.iqm_operation_mapping import map_operation
 
 
-def serialize_circuit(circuit: cirq.Circuit) -> CircuitDTO:
+def serialize_circuit(circuit: cirq.Circuit) -> Circuit:
```

Comment:
Was there a particular reason for renaming all FooDTO classes to just Foo? I think now they are confusing, especially this one, since it is easy to confuse with CIrq's Circuit class.

Reply:
I didn't think it added anything and our naming conventions in other repos are different so it was also inconsistent. The third reason is that the pulse compiler uses the objects in data processing, not just in serialization logig. Cirq on IQM uses the Circ Circuit as cirq.Circuit so if there is a risk of confusion, our Circuit could be used as iqm_client.Circuit or imported as IQMCircuit.